### ISSUE TITLE
Folded text makes the dw2pdf plugin unable to work

### ISSUE THREAD

**Issue by legofarmer** [dokufreaks/plugin-folded#70]
I use dw2pdf to export my pages to pdfs.  

Unfortunately, it doesn't work on any pages where I use folded text. The hidden plugin appears to work fine, perhaps contributors who are familiar with how that plugin is designed could use that info to better understand why the folded plugin doesn't work with it.  

If anybody else has encountered this issue and fixed it, please let it be known.  

Thanks!

**Comment by lpaulsen93** [comment]
To me this seems to be a problem in the dw2pdf plugin which is somehow triggered by the folded plugin. Exports to other formats work fine, e.g. to ODT format.  I also looked at the code for non-HTML rendering: the generated code in this case is very simple as it is exported as plain text.

But I can re-produce the issue on my local installation also. My Apache2 logs show the following error:
```
[Tue Sep 15 19:59:56.025188 2020] [php7:error] [pid 8469] [client 127.0.0.1:58780] PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 208896 bytes) in /home/lars/public-html/dokuwiki/lib/plugins/dw2pdf/vendor/mpdf/mpdf/src/Cache.php on line 42, referer: http://localhost/doku.php?id=testseite_folded
```
This tells me that the dw2pdf plugin is running in some kind of endless loop using up all memory. The issue is most likely in t he mPDF library which is used by the dw2pdf plugin.

**Comment by Klap-in** [comment]
The folded plugin wraps the syntax just with an extra div or span? I do not see yet why that should give issues. 

You might open an issue in dw2pdf plugin issue tracker. Please include a _small_ example of wiki syntax with which you have _tested_ that it was not working.

**Comment by lpaulsen93** [comment]
@legofarmer I found the problem and fixed it. Just re-install. It should work now.

**Comment by legofarmer** [comment]
> @legofarmer I found the problem and fixed it. Just re-install. It should work now.

Thank you so much! It's great!  

Do you have some time to describe the issue to itch my curiosity?

**Comment by lpaulsen93** [comment]
I did a change some days ago to fix issue #54. Unfortunately for that fix I used code from the DokuWiki core in the wrong way. This somehow caused an issue on PDF export with dw2pdf but not with normal rendering of the wiki page.

Anyway, I found a simpler solution that is working. Seems that I had a bad day as I did the change for #54. I am sorry for the hassle.
